internal/client/job.go

```diff
@@ -114,6 +123,7 @@ func (c *Project) queueAndStreamJob(
 	job *pb.Job,
 	ui terminal.UI,
 	jobIdCallback func(string),
+	mon chan pb.Job_State,
```

Comment:
This is fine, cause nothing calls `queueAndStreamJob` directly except `doJob` if I recall.